.github/action.yml

```diff
@@ -0,0 +1,20 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+
+jobs:
+  validate-and-publish:
+    name: Validate and Publish
+    runs-on: ubuntu-latest # only linux supported at present
+    steps:
+      - uses: actions/checkout@v1
+      - uses: w3c/respec-w3c-auto-publish@master # use the action
```

Comment:
Maybe use `@v1`. Master can have breaking changes. v1 is semver safe and moving release tag.

Reply:
ok, cool. Might want to update this also in other repo.